optimism.tokenlist.json

```diff
@@ -395,6 +395,50 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x03ab458634910AaD20eF5f1C8ee96F1D6ac54919",
+      "name": "Rai Reflex Index",
+      "symbol": "RAI",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/RAI.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x7FB688CCf682d58f86D7e38e03f9D22e7705448B",
+      "name": "Rai Reflex Index",
+      "symbol": "RAI",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/RAI.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 42,
+      "address": "0x76b06a2f6dF6f0514e7BEC52a9AfB3f603b477CD",
```

Comment:
We need to be able to mint test tokens here using the `0xe3a90926133fef3c58514fA1aAeF75E0998aedD5` account, see [readme](https://github.com/ethereum-optimism/ethereum-optimism.github.io#contributing-guide) in repo

Reply:
@elenadimitrova do you want to mint on Kovan or Kovan optimism? Anyone can mint RAI on Kovan using the app

https://app-kovan.reflexer.finance/

Reply:
Thanks, managed to mint on Kovan for testing 👍